test/config.js

```diff
@@ -36,20 +36,53 @@ describe('embark.Config', function () {
 
       assert.deepEqual(config.blockchainConfig, expectedConfig);
     });
+
+    it('should convert Ether units', function () {
+      let expectedConfig = {
+        "enabled": true,
+        "networkType": "custom",
+        "genesisBlock": "config/development/genesis.json",
+        "datadir": ".embark/development/datadir",
+        "isDev": false,
+        "targetGasLimit": "300000",
+        "gasPrice": "8000000",
+        "mineWhenNeeded": true,
+        "nodiscover": true,
+        "rpcHost": "localhost",
+        "rpcPort": 8545,
+        "rpcCorsDomain": "http://localhost:8000",
+        "wsOrigins": "auto",
+        "account": {
+          "password": "config/development/password"
+        }
+      };
+
+      let config = new Config({
+        env: 'unitenv',
+        configDir: './test/test1/config/',
+        events: new Events()
+      });
+      config.plugins = new Plugins({plugins: {}});
+      config.logger = new TestLogger({});
+      config.loadBlockchainConfigFile();
+
+      assert.deepEqual(config.blockchainConfig, expectedConfig);
+    });
```

Comment:
This is great!

I think I'd also like to see a test that ensures values that are not ether unit strings *still* work. E.g. If the `gasPrice` in the config is `8000000` and gets piped through the utility function, will it complain as it expects a unit string, or is it smart enough to figure out that this doesn't need to be converted?

Reason being is that we only check for `!= 'auto'` at the moment

Reply:
@PascalPrecht This was a good idea for sure! I added the test and then saw the `getWeiBalanceFromString` util function was returning hex values. I took a look at it and think this may be unexpected behavior - changing it to match the web3 `toWei` function behavior works as I would expect and matches the web3 `toWei` API spec (returns a number string or BigNumber)